lighthouse-core/test/results/artifacts/artifacts.json

```diff
@@ -1839,6 +1839,26 @@
       "detector": "js",
       "id": "wordpress",
       "name": "WordPress"
+    },
+    {
```

Comment:
I don't think we want to artificially add these to sample_v2 (we can wait for the follow up smoke test). `yarn update:sample-artifacts Stacks` (then `yarn update:sample-json` again) should update this file to what's really in dbw_tester (should be jquery + jquery fast path + wordpress)

Reply:
Ah done, removed and re-updated